ocdskingfisherprocess/checks.py

```diff
@@ -186,13 +187,24 @@ def _check_release_row(self, release_row, override_schema_version=''):
             with self.database.get_engine().begin() as connection:
                 connection.execute(self.database.release_check_table.insert(), checks)
         except APIException as err:
-            checks = [{
-                'release_id': release_row.id,
-                'error': str(err),
-                'override_schema_version': override_schema_version
-            }]
-            with self.database.get_engine().begin() as connection:
-                connection.execute(self.database.release_check_error_table.insert(), checks)
+            # This is a specific exception throw by the library.
+            # We save it to the database
+            self._store_release_row_error(release_row, override_schema_version, err)
+        except Exception as err:
+            # This is any old exception
+            # We save it to the database
+            self._store_release_row_error(release_row, override_schema_version, err)
+            # But lets also report this to Sentry, as it may be a system/host error that we should be alerted about
+            capture_exception(err)
+
+    def _store_release_row_error(self, release_row, override_schema_version, err):
```

Comment:
I'd put this next to the `_store_record_row_error` method, for easier comparison when reading or refactoring.

Reply:
Done